### ISSUE TITLE
Zero mAP when testing

### ISSUE THREAD

**Issue by rkruegs123** [jwyang/faster-rcnn.pytorch#689]
Similar to #671 I am getting all zeros for test results, but there is no answer as to why in that post. Like this previous post, it seems as though i am getting non-zero mAP from manual insepction and from the training loss.

Can anyone explain how they might have fixed it?

**Comment by Karthik-Suresh93** [comment]
hey @rkruegs123 , any update on this? I too am facing this issue. The training loss decreases well, but when I run test_net.py I get 0 mAP for all classes

**Comment by Kuuuo** [comment]
I met the same issue , have you solved it ?

**Comment by JiashengHong** [comment]
Do your labels have space or upper cases? 
In pascal_voc.py, `cls = self._class_to_ind[obj.find('name').text.lower().strip()]`
This code causes your labels mismatch to classes.
Maybe you should modify 2 places:
1. `self._classes = (...)` Make sure classes are same as your labels.
2. `cls = self._class_to_ind[obj.find('name').text]`
